tests/library/dbg/tests/test_command_branch.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import pytest
+
+from ....host import Controller
+from . import break_at_sym
+from . import get_binary
+from . import launch_to
+from . import pwndbg_test
+
+CONDBR_X64_BINARY = get_binary("conditional_branch_breakpoints_x64.out")
+
+
+@pwndbg_test
+@pytest.mark.parametrize("binary", [CONDBR_X64_BINARY], ids=["x86-64"])
+async def test_command_break_if_x64(ctrl: Controller, binary: str) -> None:
+    """
+    Tests the chain for a non-nested linked list
+    """
+    import pwndbg
+
+    if not pwndbg.dbg.is_gdblib_available():
```

Comment:
better add decorator pytest skipIf ?

Reply:
When running in LLDB, Pwndbg is only properly initialized sometime after the test starts, as part of how we handle asynchronous interactions with the Pwndbg CLI, so `skipif` would evaluate the condition way too early to be useful. We could do `pytest.skip()` instead of returning, though.